src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1204,98 @@
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
 
+# region connected-acr TODO
+helps['acr connected-acr'] = """
+type: group
+short-summary: Manage Edge/On-Prem Container Registries with Azure Container Registries.
+"""
+
+helps['acr connected-acr create'] = """
+type: command
+short-summary: Create a connected registry for an Azure Container Registry.
+examples:
+  - name: Create a connected registry with DNS name 'edge01.contoso.internal' that has 'mycloudregistry.azurecr.io' as a parent.
+    text: |
+        az acr connecter-acr create --registry mycloudregistry.azurecr.io 
+            --name Contoso-Edge01-in-Wichita --repositories "contoso-app/* contoso-service/*"
```

Comment:
@rosanch please remove wildcard from repositories. For MVP we ask users to explicitly list each repository